bedrock/contentful/templates/includes/contentful/cta.html

```diff
@@ -22,7 +22,13 @@
   ) }}
 
 {% elif action == 'Get Pocket' %}
+
     {% set referral = '?utm_source=' + utm_source + '&utm_medium=referral&utm_campaign=' + utm_campaign %}
     <a href="https://getpocket.com/{{ referral|safe }}" class="mzp-c-button {{ button_class }}" rel="external noopener" data-cta-type="link" data-cta-text="{{ cta_text }}">{{ label }}</a>
+
+{% elif action == 'Get Mozilla VPN' %}
+
+  <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button {{ button_class }}">{{ label }}</a>
```

Comment:
Suggest adding some data attributes so we can see the numbner of clicks this gets in GA?
```suggestion
  <a href="{{ url('products.vpn.landing') }}" class="mzp-c-button {{ button_class }}" data-cta-text="{{ cta_text }}" data-cta-type="button" data-cta-position="{{ location }}">{{ label }}</a>
```

Reply:
Thanks! I literally just copied the code from the current VPN CTA on the home page.

Reply:
I do see above this one that the `data-cta-type` for Pocket is `link` and not `button`. They're both buttons though. They should be the same type right?

Reply:
Good question. We seem to use these pretty interchangably and there's no real definition for `link` as a value in our docs: https://bedrock.readthedocs.io/en/latest/analytics.html#gtm-listeners-data-attributes

To err on the side of consistency I'd use `link` as well here. Although throughout the site as a whole I see a good mixture of both.

Reply:
Cool. Done.